packages/helm-charts/oracle/templates/statefulset.yaml

```diff
@@ -71,6 +74,11 @@ spec:
               key: token
               name: {{ $e }}
         {{ end }}
+        - name: CERTIFICATE_AUTHORITY
+          valueFrom:
+            secretKeyRef:
+              key: ca.crt
+              name: {{ (index .Values.kube.serviceAccountSecretNames 0) }}
```

Comment:
I think that the cluster CA is always the same (for any service account of the cluster) and always placed automatically in `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt` ([link](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod))

Reply:
I added `automountServiceAccountToken: false` to prevent the service account token from being accessible within the oracle container, but unfortunately that property is on a pod-level and not container-level

I feel like it's worth getting the cert like this so that the oracle container is more locked down, what do you think?

Reply:
Oh I see. I did not realize on the ´automountServiceAccountToken´ impact :ok_hand: